### DISCUSSION TITLE
Download as PDF fails due to font not found.

### DISCUSSION THREAD

**Discussion by nviraj** [open-webui/open-webui#4095] [General]
# Bug Report

## Description

**Bug Summary:**
Download as PDF fails due to font not found. Not sure if this error is due to me installing open webui in a conda environment via pip, but if I try to download a thread I get the following error:

> File "d:\Apps\PortableApps\open-webui\.conda\Lib\site-packages\apps\webui\routers\utils.py", line 70, in download_chat_as_pdf
> pdf.add_font("NotoSans", "", f"{FONTS_DIR}/NotoSans-Regular.ttf")
> File "d:\Apps\PortableApps\open-webui\.conda\Lib\site-packages\fpdf\fpdf.py", line 1859, in add_font
> raise FileNotFoundError(f"TTF Font file not found: {fname}")
> FileNotFoundError: TTF Font file not found: ./static/fonts/NotoSans-Regular.ttf

The fonts actually exist in the env btw
![image](https://github.com/user-attachments/assets/46365ffd-1095-45ed-9538-1945a73f1ddd)

**Steps to Reproduce:**
Download a chat as PDF

**Expected Behavior:**
Lets me export a chat as PDF

**Actual Behavior:**
Fails silently on the UI

## Environment

- **Open WebUI Version:** 0.3.8
- **Ollama (if applicable):** 0.2.8

- **Operating System:** Windows 11
- **Browser (if applicable):** Firefox 128.0

## Reproduction Details

**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Installation Method
Manual install via pip in a conda environment

**Comment by NeilSCGH** [comment]
Hello, 
I've posted a detailed version of this issue [here](https://github.com/open-webui/open-webui/issues/4236), with a workaround at the end.
